cli/src/psp-utils/constants.ts

```diff
@@ -6,8 +6,8 @@ export const PROVER_JS_VERSION = "file:../light-prover.js";
 export const PSP_DEFAULT_PROGRAM_ID =
   "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS";
 export const LIGHT_SYSTEM_PROGRAMS_VERSION =
-  '{ git = "https://github.com/lightprotocol/light-protocol", features = ["cpi"], branch = "main" }';
+  '{ git = "https://github.com/lightprotocol/light-protocol", features = ["cpi"], branch = "vadorovsky/multiple-event-merkle-trees" }';
 export const LIGHT_MACROS_VERSION =
-  '{ git = "https://github.com/lightprotocol/light-protocol", branch = "main" }';
+  '{ git = "https://github.com/lightprotocol/light-protocol", branch = "vadorovsky/multiple-event-merkle-trees" }';
 export const LIGHT_VERIFIER_SDK_VERSION =
-  '{ git = "https://github.com/lightprotocol/light-protocol", branch = "main" }';
+  '{ git = "https://github.com/lightprotocol/light-protocol", branch = "vadorovsky/multiple-event-merkle-trees" }';
```

Comment:
same for these they should be switched to main right?

Reply:
And here - not really - if I switch this one to main, PSPs will try to use the current main (without this PR). So we can switch after this PR is merged.